src/Catty/ViewController/Download/StoreProjectDownloader.swift

```diff
@@ -52,15 +52,41 @@ final class StoreProjectDownloader: StoreProjectDownloaderProtocol {
             let handleDataTaskCompletion: (Data?, URLResponse?, Error?) -> (items: StoreProjectCollection.StoreProjectCollectionNumber?, error: StoreProjectDownloaderError?)
             handleDataTaskCompletion = { data, response, error in
                 if let error = error as NSError?, error.domain == NSURLErrorDomain && error.code == NSURLErrorTimedOut {
+                    let userInfo = ["projectName": searchTerm,
+                                    "url": indexURL.absoluteURL,
+                                    "error": error.localizedDescription] as [String: Any]
+
+                    NotificationCenter.default.post(name: .projectSearchFailure, object: self, userInfo: userInfo)
```

Comment:
The `userInfo` dictionary is something very specific for the Crashlytics domain. Whenever possible try to use generic objects here at this case since these notifications can be reused for other domains as well. Also, as mentioned above, the passed object is of type `StoreProjectDownloader`, not `ProjectLoadingInfo` as assuming in the Firebase extension. Can you reuse `ProjectFetchFailureInfo` and pass it as a object?